backend/app/site_settings/default_settings.yml

```
@@ -115,3 +115,10 @@ chat:
     data_type: bool
     description: "Enable post verification for chats from js widgets."
     client: true
+
+upload:
+  upload_max_body_size:
+    default: 10485760    # 10MB
```

```suggestion
    default: 10485760    # 10 MiB
```
🤣  Sorry, MiB (Mebibyte) seems to be more correct than MB (Megabyte).